<launch>
    <arg name="robot_name" default="turtlebot2i"/>
    <arg name="shannon" default="false"/>
    <arg name="output" default="screen"/>

    <node name="network_monitor" pkg="turtlebot2i_edge" type="network_monitor.py" ns="$(arg robot_name)"
          output="$(arg output)">
        <param name="shannon" type="bool" value="$(arg shannon)"/>
        <remap from="edge/stamp" to="edge/upload"/>
    </node>
</launch>